setup.py

```diff
@@ -397,30 +392,38 @@ def run(self):
         distutils.command.clean.clean.run(self)
 
 
-setup(
-    # Metadata
-    name=package_name,
-    version=version,
-    author='PyTorch Core Team',
-    author_email='soumith@pytorch.org',
-    url='https://github.com/pytorch/vision',
-    description='image and video datasets and models for torch deep learning',
-    long_description=readme,
-    license='BSD',
-
-    # Package info
-    packages=find_packages(exclude=('test',)),
-    package_data={
-        package_name: ['*.dll', '*.dylib', '*.so']
-    },
-    zip_safe=False,
-    install_requires=requirements,
-    extras_require={
-        "scipy": ["scipy"],
-    },
-    ext_modules=get_extensions(),
-    cmdclass={
-        'build_ext': BuildExtension.with_options(no_python_abi_suffix=True),
-        'clean': clean,
-    }
-)
+if __name__ == "__main__":
+    print("Building wheel {}-{}".format(package_name, version))
+
+    write_version_file()
+
+    with open('README.rst') as f:
+        readme = f.read()
+
+    setup(
+        # Metadata
+        name=package_name,
+        version=version,
+        author='PyTorch Core Team',
+        author_email='soumith@pytorch.org',
```

Comment:
unrelated: @soumith do we want to change this e-mail to a default pytorch e-mail? PyTorch [now uses `packages@pytorch.org`](https://github.com/pytorch/pytorch/blob/9e0102c10f57631000ebf28e55a055f480b9b780/setup.py#L916)